OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -61,27 +62,41 @@ typealias Logger = OptimoveCore.Logger
                     throw GuardError.custom("Failed on OptimobileSDK initialization. Reason: \(error.localizedDescription)")
                 }
             }
+        }   
+
+        if config.isPreferenceCenterConfigured() {
+            shared.container.resolve { serviceLocator in
+                do {
+                    try OptimovePreferenceCenter.initialize(config: config, storage: serviceLocator.storage(), networkClient: NetworkClientImpl())
+                } catch {
+                    throw GuardError.custom("Failed on PreferenceCenter initialization. Reason: \(error.localizedDescription)")
+                }
+            }
         }
     }
 
     /// Set the credentials for the Optimove server. Intent to use as a step for the delayed initialization.
-    public static func setCredentials(optimoveCredentials: String?, optimobileCredentials: String?) {
+    @available(iOS 13.0, *)
+    public static func setCredentials(optimoveCredentials: String?, optimobileCredentials: String?, preferenceCenterCredentials: String?) {
```

Comment:
did we just casually break apps for all existing clients using delayed init? this requires 3rd argument. to not require it need to pass default = nil but better to create overload like in android